README.md

```diff
@@ -69,12 +70,24 @@ Keys:
 server.addr	SERVER_ADDR	address the server should bind to	(:80)
```

Comment:
markdown blockquote `bash` helper provides strange results when used to represent shell responses.

```bash
 $ ./a.out --help
Keys:
 server.addr	SERVER_ADDR	address the server should bind to	(:80)
 ```

(here bind is colored)

please consider to use `console` helper (or `shell`), that is available to represent the command and the result

 ```console
 $ ./a.out --help
 Keys:
server.addr	SERVER_ADDR	address the server should bind to	(:80)
```